<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="IT3786-kirankumar" name="paymentsUI.Payment_NodeJS_CC_PG_Amex" tests="1" failures="1" timestamp="25 Feb 2020 09:47:19 GMT" time="95.692" errors="0">
  <testcase name="paymentNodeJSCCAmex" time="95.692" classname="paymentsUI.Payment_NodeJS_CC_PG_Amex">
    <failure type="java.lang.AssertionError" message="Failure! Element not Visible!">
      <![CDATA[java.lang.AssertionError: Failure! Element not Visible!
at org.testng.AssertJUnit.fail(AssertJUnit.java:59)
at org.testng.AssertJUnit.assertTrue(AssertJUnit.java:24)
at common.WrapperMethod.elementPresent(WrapperMethod.java:1004)
at common.WrapperMethod.safeClick(WrapperMethod.java:1215)
at domains.PaymentNodeJS.paymentNodeJS_BankPage(PaymentNodeJS.java:546)
at domains.PaymentNodeJS.paymentNodeJS_Make_Payment(PaymentNodeJS.java:315)
at paymentsUI.Payment_NodeJS_CC_PG_Amex.paymentNodeJSCCAmex(Payment_NodeJS_CC_PG_Amex.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- paymentNodeJSCCAmex -->
</testsuite> <!-- paymentsUI.Payment_NodeJS_CC_PG_Amex -->
